src/components/slider/slider.ts

```diff
@@ -0,0 +1,250 @@
+import {
+  Component,
+  ElementRef,
+  HostBinding,
+  Input,
+  ViewEncapsulation,
+  AfterContentInit,
+} from '@angular/core';
+
+@Component({
+  moduleId: module.id,
+  selector: 'md-slider',
+  host: {
+    'tabindex': '0',
```

Comment:
@jelbourn Any reason we're bringing back `host` instead of adding them as `@HostBinding()` and `@HostListener`?